python/sglang/srt/layers/moe/fused_moe_triton/layer.py

```diff
@@ -627,3 +633,44 @@ def make_expert_params_mapping(
                 ("w3", ckpt_up_proj_name),
             ]
         ]
+
+
+class FlashInferFusedMoE(FusedMoE):
```

Comment:
cc @ch-wan who IIRC is doing refactors about all MoEs - does this conflict with your refactors?

Reply:
This code is simple. We can leave it as now.